config/default.yml

```diff
@@ -11,6 +11,13 @@ Performance/AncestorsInclude:
   Safe: false
   VersionAdded: '1.7'
 
+Performance/ArrayPushSingle:
+  Description: 'Use `array << x` instead of `array.push(x)`.'
+  Reference: 'https://github.com/fastruby/fast-ruby#ancestorsinclude-vs--code'
+  Enabled: true
```

Comment:
Should it be `true`, `false` or `pending`?

Reply:
The new cop generator should output pending. On the next major release all pending cops are enabled in bulk according to the [docs](https://docs.rubocop.org/rubocop/versioning.html#pending-cops).